ports/matplotplusplus/vcpkg.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "matplotplusplus",
+  "version-date": "2021-04-11",
+  "description": "A C++ graphics library for data visualization",
+  "homepage": "https://alandefreitas.github.io/matplotplusplus/",
+  "license": "MIT",
+  "supports": "!uwp",
```

```suggestion
  "supports": "!(arm | uwp)",
```
Seems it also doesn't support arm.